tests/rados/test_osd_inconsistency_pg.py

```diff
@@ -0,0 +1,110 @@
+"""
+The script is to generate and verification of the inconsistent pg
+"""
+
+import random
+import time
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.rados_test_util import create_pools
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    client_node = ceph_cluster.get_nodes(role="client")[0]
+    pool_obj = PoolFunctions(node=cephadm)
+    try:
+        log.info("Creating the pool replicated pool")
+        create_pools(config, rados_obj, client_node)
+        log.info("Pool creation completed")
+        pool_name = config["create_pools"][0]["create_pool"]["pool_name"]
+
+        # Adding an object
+        oname = "UNIQUEOBJECT{i}".format(i=random.randint(0, 10000))
+        log.info(f"Creating the object-{oname} in the pool-{pool_name}")
+        pool_obj.do_rados_put(client=client_node, pool=pool_name, obj_name=oname)
+        log.info(f"The object-{oname} is created in the pool-{pool_name}")
+        osd_map_output = rados_obj.get_osd_map(pool=pool_name, obj=oname)
+        primary_osd = osd_map_output["acting_primary"]
+        log.info(f"The object stored in the primary osd number-{primary_osd}")
+        pg_id = osd_map_output["pgid"]
+        log.info(f"The object {oname} is created in the pg-{pg_id}")
+        # Adding the key value to object
```

Comment:
Please write a method for adding a key value to the object so that we can reuse the same in other tests for inconsistency.

Reply:
Inconsistent objects are created with the different procedure.The methods are added.